src/lib/genesis_ledger_helper/lib/genesis_ledger_helper_lib.ml

```diff
@@ -645,3 +624,193 @@ let runtime_config_of_precomputed_values (precomputed_values : Genesis_proof.t)
     ; ledger = None
     ; epoch_data = None
     }
+
+(*
```

Comment:
remove the commented code?

Reply:
think this was removed in this commit: https://github.com/MinaProtocol/mina/pull/12621/commits/19fe0529586b2b4b9faaac3df4304515ec146948